schemas/json/font-collection.json

```
@@ -2,6 +2,99 @@
 	"title": "JSON schema for WordPress Font Collections",
 	"$schema": "http://json-schema.org/draft-04/schema#",
 	"type": "object",
+	"definitions": {
+		"fontFace": {
+			"description": "Font face settings. The same as theme.json but with the preview key",
+			"type": "object",
+			"properties": {
+				"preview": {
+					"description": "URL to a preview image of the font",
```

Nitpick: most of the other descriptions have periods.
```suggestion
					"description": "URL to a preview image of the font.",
```